applications/crossbar/doc/channels.md

```diff
@@ -111,3 +111,14 @@ curl -v -X POST \
     -d '{"data": {"action": "transfer", "target": "2600", "takeback_dtmf": "*1", "moh": "media_id" }}' \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/channels/{UUID}
 ```
+
+#### Put a feature on a channel
```

Comment:
Can you add a small description? For instance, what types of features are available?

Also, can you add example JSON payloads for the request?